sdk-libs/program-test/README.md

```
@@ -0,0 +1,133 @@
+<!-- cargo-rdme start -->
+
+## Light Program Test
```

markdownlint-cli2 (0.20.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)